interfaces/builtin/basedeclaration.go

```diff
@@ -154,6 +154,8 @@ plugs:
   snapd-control:
     allow-installation: false
     deny-auto-connection: true
+  unity8:
+    allow-installation: false
```

Comment:
There is an "added test" for that already -- in the form of adding "unity8" to an interface list in the TestPlugInstallation function (near line 512).  I didn't add any new functions though, true.

The only test that snapd-control has that unity8 doesn't is around its "deny-auto-connection: true" plug declaration.  But unity8 doesn't have that (and doesn't want that, right?).